neko/mod.js

```diff
@@ -1,42 +1,46 @@
 let logger = null
 let store = null
-
+let imgs = []
+const fs = require('fs')
+const path = require('path')
+let options = []
+fs.readdir(path.join(__dirname, "imgs"), (err, files) => {
+  files.forEach(file => {
+    let assetPath = "assets://mod/neko/imgs/" + file
+    imgs.push(assetPath)
+    options.push({value: assetPath, label: path.parse(file).name})
+  })
+})
 module.exports = {
   title: "Neko",
   summary: "gives you a little friend",
   author: "GiovanH",
-  modVersion: 0.2,
+  modVersion: 0.3,
 
   trees: {
     "./": "assets://mod/neko/"
   },
-
+  
   settings: {
-    radio: [{
-      model: "image",
-      label: "Image",
-      options: [{
-        value: "assets://mod/neko/vriskaimg.png",
-        label: "Vriska",
-      },{
-        value: "assets://mod/neko/dave.png",
-        label: "Babydave",
-      },{
-        value: "assets://mod/neko/nepeta.png",
-        label: "Nepeta",
-      }]
-    }],
     boolean: [{
+      model: "random-image",
+      label: "Randomize Image"
+    }, {
       model: "fixedpos",
       label: "Fixed position",
-      desc: "Attach to your screen, not the page.",
-    }]
+      desc: "Attach to your screen, not the page."
+    }],
+    radio: [{
+      model: "image",
+      label: "Image"
+    }],
   },
 
   computed(api) {
-    logger = api.logger
-    store = api.store
-    store.set("image", store.get("image", "assets://mod/neko/vriskaimg.png"))
+    logger = api.logger	
```

Comment:
We probably don't need the logger anymore on this file. Not hurting anything it's just not being used